### DISCUSSION TITLE
v4.0.1

### DISCUSSION THREAD

**Discussion by ShankarWarang** [Edgeware-Network/edgeware-node#38] [Announcements]
### New changes since v4.0.0:
- Ported consensus transition logic to the new substrate convention. {[Block #14555555](https://edgeware.subscan.io/block/14555555) will be the switch. It should ideally happen after the runtime upgrade v4.0.1 on the mainnet.} [Huge thanks to KDPnetstake for finding the critical node client bug on the previous Beresheet v4 testnet.]
- Amended the testnet block trigger to span two eras with the old consensus before switching to the new one.

<hr /><em>This discussion was created from the release <a href='https://github.com/edgeware-network/edgeware-node/releases/tag/v4.0.1'>v4.0.1</a>.</em>

**Comment by ramavats** [comment]
Thanks to KDPnetstake <3
